opendps/Makefile

```diff
@@ -80,6 +82,7 @@ OBJS = \
     ili9163c.o \
     mini-printf.o \
     gfx_lookup.o \
+    font-settings.o \
```

Comment:
Maybe this should be called something like "font full" as we might use it in other places that the settings page.

Reply:
Yeah, makes sense.
How about we split them up to "full" and "meter" fonts, so that we end up with say:

    font-small    => font-meter-small
    font-medium   => font-meter-medium
    font-large    => font-meter-large
    font-settings => font-full-small

Reply:
Ah, great idea to rename the "meter" fonts.

Reply:
There we go.